spec/models/assignment_spec.rb

```diff
@@ -14,249 +14,425 @@
                                 description_url: 'https://expertiza.ncsu.edu/', round: 1)
   end
   let(:topic_due_date) { build(:topic_due_date, deadline_name: 'Submission', description_url: 'https://github.com/expertiza/expertiza') }
-
   describe '.max_outstanding_reviews' do
-    it 'returns 2 by default'
+    it 'returns 2 by default' do
+      expect(Assignment.max_outstanding_reviews).to equal(2)
+    end
   end
-
   describe '#team_assignment?' do
-    it 'checks an assignment has team'
+    it 'checks an assignment has team' do
+      expect(assignment.team_assignment).to eql(true)
+    end
   end
-
-  describe '#has_topics?' do
+  # Need to create assignment, else giving error
+  describe '#topics?' do
     context 'when sign_up_topics array is not empty' do
-      it 'says current assignment has topics'
+      it 'says current assignment has topics' do
+        @assignment = create(:assignment)
```

Comment:
I saw you use `@assignment = create(:assignment` a lot. You can use variable `assignment` directly and without creating a new assignment since I mock an assignment in line 2.